packages/jest/examples/simple-app/package.json

```diff
@@ -7,40 +7,39 @@
     "build": "yarn ng build",
     "test": "yarn ng test",
     "lint": "yarn ng lint",
-    "e2e": "yarn ng e2e",
-    "postinstall": "ngcc"
```

Comment:
Why the `ngcc` postinstall has been removed?

Reply:
postinstall is no longer recommended (it it faster to process libs that application will import than whole node_modules)
https://github.com/angular/angular/blob/master/CHANGELOG.md#910-2020-03-25

Reply:
See my other comment